exercises/risk-risiko.cpp

```diff
@@ -27,3 +27,91 @@
   M 3 vs 3 => blue win
   O 2 vs 1 => red win
 */
+#include <iostream>
+using namespace std;
+class Dice
+{
+public:
+  int value;
+  Dice(int n)
+  {
+    value = (rand() % n) + 1;
+  }
+};
+
+int current_time_nanoseconds()
+{
+  struct timespec tm;
+  clock_gettime(CLOCK_REALTIME, &tm);
+  return tm.tv_nsec;
+}
+
+int getMaxIndex(Dice *dice_array)
+{
+  int max, k;
+  max = k = 0;
+  for (int i = 0; i < 3; i++)
```

Comment:
quel 3 potrebbe essere messo in un bel define, così per cambiarlo la prossima volta dovrai solo cambiare un'unica variabile e non tutto il codice, per il resto approvo la PR